### DISCUSSION TITLE
Change Port from CLI?

### DISCUSSION THREAD

**Discussion by black-jmyntrn** [bitfocus/companion#2625] [Q&A] [answered]
how would one change the port for companion from the cli? on every new install I never see the initial screen to set the port on my raspberry pi4. I am trying to use little backup box on the same device and that also uses port 8000 only its handled using apache2.

as of now, in the web gui, there is nowhere to change te 8000 port number to something else.

**Comment by Julusian** [comment, marked as answer]
Companion is being run as a system service on companion-pi, so is lacking the 'launcher' ui.

Changing the port isn't currently exposed, no-one else has asked for it.  
You could do it by modifying the line https://github.com/bitfocus/companion-pi/blob/main/launch.sh#L16 (at `/usr/local/src/companion-pi/launch.sh` ) and adding ` --admin-port 8080` at the end of the line (substituting in the port number you choose)  
But this might cause future updates to the companion-pi tooling to break, when/if this line gets changed by us and results in a merge conflict

**Reply by black-jmyntrn** [reply]
ohhh thank you! now all is working!

**Reply by black-jmyntrn** [reply]
so, this doesnt work anymore, i don't have that file after a fresh install. Im trying to run this along side another app that uses 8000 and the lack of being able to change the port number as someone might just want to run other things that might just use 8000 is just bonkers to me.

**Reply by black-jmyntrn** [reply]
ahhh.. its now companionpi/ versus companion-pi/
